services/capture/src/main/java/ca/bc/gov/tno/services/capture/config/CaptureConfig.java

```diff
@@ -81,7 +81,7 @@ public CaptureConfig(DataSource dataSource) {
     setMaxFailedAttempts(dataSource.getRetryLimit());
     var connection = dataSource.getConnection();
 
-    setAudioUrl((String) connection.get("audioUrl"));
+    setAudioUrl((String) connection.get("url"));
```

Comment:
I don't believe this is valid.  It should be `audioUrl` based on the original information you provided.  Additionally, the UI is mapping to `audioUrl` too.  

I actually prefer the more generic `url`.  So if you would like to use it, then you should also update the frontend to use that value too.

![image](https://user-images.githubusercontent.com/3180256/164756164-5e5bdb78-30b2-43e2-8297-a8a14fc37686.png)

Reply:
The only reason I changed this was because the front end is saving "url" in the connection field. Are you saying the front end should save the url in the field "audioUrl"?

Reply:
The frontend label says **Stream URL**.  But the json property is `audioUrl` because that is what you originally where expecting.

![image](https://user-images.githubusercontent.com/3180256/164788635-7ab78e84-65de-4a62-955f-d6e78e32cdc1.png)

Reply:
I'll look at this when I get the TNO environment running again. Having used the new UI to create the connection string for a  data source I noticed it stored the audioUrl value in the json attribute "url" which is why I made this change. I can confirm later.

Reply:
It looks like I was looking at an RSS connection string when I saw the "url" attribute name. Now that I can test it I see that the News Radio url is named "audioUrl" so I'll set it back. The other thing that convinced me it should be named "url" is that  you updated the initConfigs() method of the Capture service to include this code:

var approvedDataSources = dataSources.stream()
        .filter((ds) -> ds.getIsEnabled() &&
            ds.getConnection().get("url") != null)
        .toList();

This failed to add any of my data sources to approvedDataSources because the attribute was named "audioUrl". I'll update this code to include "audioUrl" in place of "url".

Reply:
You can change to `url`.  I prefer it too.  But you'll need to change all relevant code so that it works throughout.

Reply:
The front-end code will need to be changed. Do you think I have the necessary knowledge to do that?

Reply:
Should be as simple as search and replace `audioUrl` with `url`.  Then you can test locally first.

Reply:
Ok, I'll do that.